### DISCUSSION TITLE
After nvchadupdate I keep getting fzf related error

### DISCUSSION THREAD

**Discussion by ayushxx7** [NvChad/NvChad#2454] [Q&A] [answered]
```bash
Failed to run `config` for telescope.nvim
...a/lazy/telescope.nvim/lua/telescope/_extensions/init.lua:10: 'fzf' extension doesn't exist or isn't installed: ...nvim-data/lazy/telescope-fzf-native.nvim/lua/fzf_lib.lua:11: cannot load module 'C:/Users/ayush/AppData/Local/nvim-data/lazy/telescope-fzf-native.nvim/lua/../build/libfzf.dll': The specified module could not be found.^M
# stacktrace:
  - telescope.nvim\lua\telescope\_extensions\init.lua:10 _in_ **load_extension**
  - telescope.nvim\lua\telescope\_extensions\init.lua:62 _in_ **load_extension**
  - ~\AppData\Local\nvim/lua/plugins/init.lua:247 _in_ **config**
  - vim\_editor.lua:341 _in_ **cmd**
  - ui\lua\nvchad\nvdash\init.lua:138
 ```

I am on windows. 

I tried the solution of going to "fzf native" folder and running "make clean" / "make" but it doesn't resolve my issue :( 

ref - https://github.com/AstroNvim/AstroNvim/issues/58

**Comment by ayushxx7** [comment, marked as answer]
I resolved this by manually downloading the lib file and pasting it into `C:\Users\ayush\AppData\Local\nvim-data\lazy\telescope-fzf-native.nvim\build`.
Downloaded https://github.com/airtonix/telescope-fzf-native.nvim/releases/download/0.0.2/windows-x64-cc-libfzf.dll from https://github.com/airtonix/telescope-fzf-native.nvim/releases/tag/0.0.2

**Reply by siduck** [reply]
@ayushxx7 you only need make installed on windwos
